packages/gatsby-plugin-google-analytics/README.md

```diff
@@ -88,6 +90,10 @@ If you enable this optional option, Google Analytics will not be loaded at all f
 
 If you need to exclude any path from the tracking system, you can add it (one or more) to this optional array as glob expressions.
 
+## The "enabled" option
+
+By default, tracking is disabled in development mode and enabled in production. If you want to fully disable/enable tracking in both environments, set this to `true` or `false`. To track in production but not development, omit the option or set to `'auto'`.
```

Comment:
It would be good idea to add warning/note about using this in development (polluting analytics log) and provide sample setup that can be used to test this without polluting logs for production